### PR TITLE
2024 update calendar

### PR DESCRIPTION
**Pull request by ryjones**
Adding updates for 2024

### CODE REVIEW

**Comment by ryjones** [comment]
@tkuhrt plz review

File: project-reports/2024/2024-updated.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+| Q1      | 2024-03-21 | Besu      |
+| Q1      | 2024-03-21 | Caliper   |
+| Q2      | 2024-04-11 | Cacti     |
+| Q2      | 2024-04-11 | Fabric    |
+| Q2      | 2024-04-25 | Sawtooth  |
+| Q2      | 2024-05-02 | Aries     |
+| Q2      | 2024-05-02 | Indy      |
+| Q2      | 2024-05-02 | AnonCreds |
+| Q2      | 2024-05-09 | Iroha     |
+| Q2      | 2024-05-16 | Bevel     |
```
Comment (tkuhrt) [inline project-reports/2024/2024-updated.md:36]:
```suggestion
| Q2      | 2024-05-16 | Bevel     |
| Q2      | 2024-05-23 | Transact  |
| Q2      | 2024-05-23 | Solang    |
```

File: project-reports/2024/2024-updated.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+| Q2      | 2024-06-20 | Caliper   |
+| Q3      | 2024-07-11 | Cacti     |
+| Q3      | 2024-07-11 | Fabric    |
+| Q3      | 2024-07-25 | Sawtooth  |
+| Q3      | 2024-08-01 | Aries     |
+| Q3      | 2024-08-01 | Indy      |
+| Q3      | 2024-08-01 | AnonCreds |
+| Q3      | 2024-08-08 | Iroha     |
+| Q3      | 2024-08-15 | Bevel     |
+| Q3      | 2024-08-22 | Solang    |
```
Comment (tkuhrt) [inline project-reports/2024/2024-updated.md:52]:
```suggestion
| Q3      | 2024-08-22 | Transact  |
| Q3      | 2024-08-22 | Solang    |
```

File: project-reports/2024/2024-updated.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+| Q3      | 2024-09-05 | Firefly   |
+| Q3      | 2024-09-19 | Besu      |
+| Q3      | 2024-09-19 | Caliper   |
+| Q4      | 2024-10-10 | Cacti     |
+| Q4      | 2024-10-10 | Fabric    |
+| Q4      | 2024-10-24 | Sawtooth  |
+| Q4      | 2024-11-07 | Aries     |
+| Q4      | 2024-11-07 | Indy      |
+| Q4      | 2024-11-07 | Iroha     |
+| Q4      | 2024-11-07 | AnonCreds |
```
Comment (tkuhrt) [inline project-reports/2024/2024-updated.md]:
```suggestion
| Q4      | 2024-11-07 | AnonCreds |
| Q4      | 2024-11-07 | Iroha     |
```

File: project-reports/2024/2024-updated.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+| Q4      | 2024-11-14 | Bevel     |
+| Q4      | 2024-11-21 | Solang    |
```
Comment (tkuhrt) [inline project-reports/2024/2024-updated.md:66]:
```suggestion
| Q4      | 2024-11-21 | Transact  |
| Q4      | 2024-11-21 | Solang    |
```

File: project-reports/2024/2024-updated.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+| Q4      | 2024-11-28 | Cello     |
+| Q4      | 2024-12-05 | Besu      |
+| Q4      | 2024-12-05 | Caliper   |
+| Q4      | 2024-12-05 | Firefly   |
```
Comment (tkuhrt) [inline project-reports/2024/2024-updated.md]:
```suggestion
| Q4      | 2024-12-05 | Firefly   |
| Q4      | 2024-12-05 | Besu      |
| Q4      | 2024-12-05 | Caliper   |
```

File: project-reports/2024/2024-updated.md [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+has_children: false
+has_toc: false
+---
+
+# 2024 Schedule
+
+
+| Quarter | Date       | Project   |
+| ------- | ---------- | --------- |
+| Q1      | 2024-01-10 | Cacti     |
```
Comment (tkuhrt) [inline project-reports/2024/2024-updated.md]:
```suggestion
| Q1      | 2024-01-11 | Cacti     |
```

File: project-reports/2024/2024-updated.md [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+| Q1      | 2024-01-10 | Fabric    |
```
Comment (tkuhrt) [inline project-reports/2024/2024-updated.md]:
```suggestion
| Q1      | 2024-01-11 | Fabric    |
```

File: project-reports/2024/2024-updated.md [hunk 8]
```diff
@@ -0,0 +1,66 @@
...
+| Q1      | 2024-01-24 | Sawtooth  |
```
Comment (tkuhrt) [inline project-reports/2024/2024-updated.md]:
```suggestion
| Q1      | 2024-01-25 | Sawtooth  |
```

File: project-reports/2024/2024-updated.md [hunk 9]
```diff
@@ -0,0 +1,66 @@
...
+| Q1      | 2024-01-31 | Aries     |
+| Q1      | 2024-01-31 | Indy      |
+| Q1      | 2024-01-31 | AnonCreds |
```
Comment (tkuhrt) [inline project-reports/2024/2024-updated.md]:
```suggestion
| Q1      | 2024-02-01 | Aries     |
| Q1      | 2024-02-01 | Indy      |
| Q1      | 2024-02-01 | AnonCreds |
```

File: project-reports/2024/2024-updated.md [hunk 10]
```diff
@@ -0,0 +1,66 @@
...
+| Q1      | 2024-02-07 | Iroha     |
```
Comment (tkuhrt) [inline project-reports/2024/2024-updated.md]:
```suggestion
| Q1      | 2024-02-08 | Iroha     |
```

File: project-reports/2024/2024-updated.md [hunk 11]
```diff
@@ -0,0 +1,66 @@
...
+| Q1      | 2024-02-14 | Bevel     |
```
Comment (tkuhrt) [inline project-reports/2024/2024-updated.md]:
```suggestion
| Q1      | 2024-02-15 | Bevel     |
```

File: project-reports/2024/2024-updated.md [hunk 12]
```diff
@@ -0,0 +1,66 @@
...
+| Q1      | 2024-02-21 | Transact  |
+| Q1      | 2024-02-21 | Solang    |
```
Comment (tkuhrt) [inline project-reports/2024/2024-updated.md]:
```suggestion
| Q1      | 2024-02-22 | Transact  |
| Q1      | 2024-02-22 | Solang    |
```

File: project-reports/2024/2024.csv [hunk 13]
```diff
@@ -0,0 +1,54 @@
+Quarter,Date,Project
+Q1,2024-01-10,Cacti
+Q1,2024-01-10,Fabric
```
Comment (tkuhrt) [inline project-reports/2024/2024.csv]:
```suggestion
Q1,2024-01-11,Cacti
Q1,2024-01-11,Fabric
```

File: project-reports/2024/2024.csv [hunk 14]
```diff
@@ -0,0 +1,54 @@
+Quarter,Date,Project
+Q1,2024-01-10,Cacti
+Q1,2024-01-10,Fabric
+Q1,2024-01-24,Sawtooth
```
Comment (tkuhrt) [inline project-reports/2024/2024.csv]:
```suggestion
Q1,2024-01-25,Sawtooth
```

File: project-reports/2024/2024.csv [hunk 15]
```diff
@@ -0,0 +1,54 @@
+Quarter,Date,Project
+Q1,2024-01-10,Cacti
+Q1,2024-01-10,Fabric
+Q1,2024-01-24,Sawtooth
+Q1,2024-01-31,Aries
+Q1,2024-01-31,Indy
+Q1,2024-01-31,AnonCreds
```
Comment (tkuhrt) [inline project-reports/2024/2024.csv]:
```suggestion
Q1,2024-02-01,Aries
Q1,2024-02-01,Indy
Q1,2024-02-01,AnonCreds
```

File: project-reports/2024/2024.csv [hunk 16]
```diff
@@ -0,0 +1,54 @@
+Quarter,Date,Project
+Q1,2024-01-10,Cacti
+Q1,2024-01-10,Fabric
+Q1,2024-01-24,Sawtooth
+Q1,2024-01-31,Aries
+Q1,2024-01-31,Indy
+Q1,2024-01-31,AnonCreds
+Q1,2024-02-07,Iroha
```
Comment (tkuhrt) [inline project-reports/2024/2024.csv]:
```suggestion
Q1,2024-02-08,Iroha
```

File: project-reports/2024/2024.csv [hunk 17]
```diff
@@ -0,0 +1,54 @@
+Quarter,Date,Project
+Q1,2024-01-10,Cacti
+Q1,2024-01-10,Fabric
+Q1,2024-01-24,Sawtooth
+Q1,2024-01-31,Aries
+Q1,2024-01-31,Indy
+Q1,2024-01-31,AnonCreds
+Q1,2024-02-07,Iroha
+Q1,2024-02-14,Bevel
```
Comment (tkuhrt) [inline project-reports/2024/2024.csv]:
```suggestion
Q1,2024-02-15,Bevel
```

File: project-reports/2024/2024.csv [hunk 18]
```diff
@@ -0,0 +1,54 @@
...
+Q1,2024-01-10,Cacti
+Q1,2024-01-10,Fabric
+Q1,2024-01-24,Sawtooth
+Q1,2024-01-31,Aries
+Q1,2024-01-31,Indy
+Q1,2024-01-31,AnonCreds
+Q1,2024-02-07,Iroha
+Q1,2024-02-14,Bevel
+Q1,2024-02-21,Transact
+Q1,2024-02-21,Solang
```
Comment (tkuhrt) [inline project-reports/2024/2024.csv]:
```suggestion
Q1,2024-02-22,Transact
Q1,2024-02-22,Solang
```

File: project-reports/2024/2024.csv [hunk 19]
```diff
@@ -0,0 +1,54 @@
...
+Q1,2024-03-21,Besu
+Q1,2024-03-21,Caliper
+Q2,2024-04-11,Cacti
+Q2,2024-04-11,Fabric
+Q2,2024-04-25,Sawtooth
+Q2,2024-05-02,Aries
+Q2,2024-05-02,Indy
+Q2,2024-05-02,AnonCreds
+Q2,2024-05-09,Iroha
+Q2,2024-05-16,Bevel
```
Comment (tkuhrt) [inline project-reports/2024/2024.csv]:
```suggestion
Q2,2024-05-16,Bevel
Q2,2024-05-23,Transact
Q2,2024-05-23,Solang
```

File: project-reports/2024/2024.csv [hunk 20]
```diff
@@ -0,0 +1,54 @@
...
+Q2,2024-06-20,Caliper
+Q3,2024-07-11,Cacti
+Q3,2024-07-11,Fabric
+Q3,2024-07-25,Sawtooth
+Q3,2024-08-01,Aries
+Q3,2024-08-01,Indy
+Q3,2024-08-01,AnonCreds
+Q3,2024-08-08,Iroha
+Q3,2024-08-15,Bevel
+Q3,2024-08-22,Solang
```
Comment (tkuhrt) [inline project-reports/2024/2024.csv]:
```suggestion
Q3,2024-08-22,Transact
Q3,2024-08-22,Solang
```

File: project-reports/2024/2024.csv [hunk 21]
```diff
@@ -0,0 +1,54 @@
...
+Q3,2024-09-05,Firefly
+Q3,2024-09-19,Besu
+Q3,2024-09-19,Caliper
+Q4,2024-10-10,Cacti
+Q4,2024-10-10,Fabric
+Q4,2024-10-24,Sawtooth
+Q4,2024-11-07,Aries
+Q4,2024-11-07,Indy
+Q4,2024-11-07,Iroha
+Q4,2024-11-07,AnonCreds
```
Comment (tkuhrt) [inline project-reports/2024/2024.csv]:
```suggestion
Q4,2024-11-07,AnonCreds
Q4,2024-11-07,Iroha
```

File: project-reports/2024/2024.csv [hunk 22]
```diff
@@ -0,0 +1,54 @@
...
+Q4,2024-11-14,Bevel
+Q4,2024-11-21,Solang
```
Comment (tkuhrt) [inline project-reports/2024/2024.csv]:
```suggestion
Q4,2024-11-21,Transact
Q4,2024-11-21,Solang
```

File: project-reports/2024/2024.csv [hunk 23]
```diff
@@ -0,0 +1,54 @@
...
+Q4,2024-11-28,Cello
+Q4,2024-12-05,Besu
+Q4,2024-12-05,Caliper
+Q4,2024-12-05,Firefly
```
Comment (tkuhrt) [inline project-reports/2024/2024.csv]:
```suggestion
Q4,2024-12-05,Firefly
Q4,2024-12-05,Besu
Q4,2024-12-05,Caliper
```

**Review by tkuhrt — CHANGES_REQUESTED** [review]

**Comment by ryjones** [comment]
@tkuhrt if this is ready, go ahead and merge

**Review by tkuhrt — APPROVED** [review]

**Pull request merged**